Makefile

```diff
@@ -51,9 +52,15 @@ BUILD_OPTS_DEPLOY?="-ldflags=$(BUILDINFO) -w -s"
 
 check: lint test ## Run linters and tests
 
-build: host-apps bin ## Build apps and binaries. `go build` with ${OPTS}
+move-built-frontend:
+	rm -rf ${MANAGER_UI_BUILT_DIR}
+	mkdir ${MANAGER_UI_BUILT_DIR}
+	cp -r ${MANAGER_UI_DIR}/dist/. ${MANAGER_UI_BUILT_DIR}
+	rm -rf ${MANAGER_UI_DIR}/dist/
 
-build-static: host-apps-static bin-static ## Build apps and binaries. `go build` with ${OPTS}
+build: move-built-frontend host-apps bin ## Install dependencies, build apps and binaries. `go build` with ${OPTS}
```

Comment:
I think having `move-built-frontend` here is a problem. With this, now the `dist` folder will have to be present every time anybody wants to build the binaries. As it was removed from the repo, that would mean the need to have all the front-end tools intalled and build the UI each time befor building the binaries, which would be inconvinient. Also, the process would be complicate even having the front-end tools, because `make build-ui` calls `make move-built-frontend`, which would remove the `dist` folder just after finishing, so a call to `make build` just after calling `make build-ui` would fail. I think the solution would be to remove `move-built-frontend` from here.

Maybe the same applies to `build-static`.

Reply:
@Senyoret1 agreed. but we definitely need `dist` in `cmd/skywire-visor` to build binaries now. native embedding requires that files being embedded are ready at compile time. I removed this command but left it in `build-ui`. so that built ui in the end lies where needed